client/signup/reskinned-processing-screen/index.jsx

```diff
@@ -26,6 +26,14 @@ const useSteps = ( { flowName, hasPaidDomain, isDestinationSetupSiteFlow } ) =>
 		case 'site-content-collection':
 			steps = [ { title: __( 'Saving your content' ) }, { title: __( 'Closing the loop' ) } ];
 			break;
+		case 'newsletter':
+			steps = [
+				{ title: __( 'Saving your preferences' ) },
+				{ title: __( 'Getting your Domain' ) },
+				{ title: __( 'Adding your Plan' ) },
+				{ title: __( 'Preparing Checkout' ) },
```

Comment:
The capitalization here is inconsistent. `preferences` vs `Domain` and `your` is lowercased. I think only the first character should be capitalized.

Reply:
I think that too, but here we just followed the designs. We should ask @saygunnyc